lib/utils.js

```diff
@@ -40,11 +40,20 @@ function wrapWithBackslash(str, options) {
     return out;
 }
 
+function removeTags(str){
```

Comment:
I don't think I understand what removeTags is for? Is it to sanitise text in case it contains HTML? If so, then we shouldn't do it with regexps; instead, insert the text with `node.appendChild(document.createTextNode(theText))` in client-side JS and then the browser does it for us, rather than us manually escaping.

Reply:
Didn't know the best way to handle it, but i agree i think adding it to client-side JS is the much better idea.

Reply:
For example, a name like `First Last/><script>alert('xss')</script>` breaks the search page without anything being searched because the script tag ends within `searchdata`.